### PR TITLE
Clearer instructions for app removal consequences (closes #320)

### LINKED ISSUE #320: Misleading FAQ information regarding uninstall with multiple apps
## Where to find the issue
https://www.coronawarn.app/en/faq/#delete_random_ids 4th paragraph or
https://github.com/corona-warn-app/cwa-website/blob/master/src/data/faq.json line 409

## Describe the issue
The FAQ says:
"On Android devices, the log is deleted if Corona-Warn-App was the last app using the framework on the device. If you have SwissCovid or Immuni installed in addition to the Corona-Warn-App, the log will only be removed if these additional apps are de-installed or you manually trigger deletion of the exposure log."

By experiment with CWA and SwissCovid installed I find that uninstalling either app and leaving the other app installed will remove all log entries which are linked to the app being uninstalled and which are displayed under
Settings > Google > COVID-19 exposure notifications > Exposure checks

So I think that the FAQ is misleading. For me the log is the history displayed as described above. In this definition of "log" the log is not fully preserved until the last COVID-19 app is uninstalled.

Also the effect of uninstalling one app is to pass on ownership of the API to the remaining app, if this app did not previously own the API. I haven't found any documentation which describes that only one app can be active towards the Google Exposure Notification System at any one time.

## Suggested change
Perhaps the FAQ should be referring to Temporary Exposure Keys and Rotating Proximity Identifiers (RPIs) (observed and sent)? The term "log" as used in the FAQ needs to be explained.
Refer to terms used on https://developers.google.com/android/exposure-notifications/exposure-notifications-api#architecture 

Also, since this is functionality from the Exposure Notification System, Google could be encouraged to elaborate their own documentation regarding these points on the [Google API for Exposure Notifications](https://developers.google.com/android/exposure-notifications/exposure-notifications-api) documentation site.

### PR DESCRIPTION
**Pull request by tkowark**
Removes the part about other apps and potential retaining of random IDs.
Adds link to Googles implementation.
Urges people to only de-install when instructed to do so.

### CODE REVIEW

**Review by MarlisFriedl — APPROVED** [review]
Looks good to me

File: src/data/faq.json [hunk 1]
```diff
@@ -407,8 +407,8 @@
...
                             "In the current version, deleting the app on iOS devices leaves the exposure log intact without any guarantees for the duration of storage.",
-                            "On Android devices, the log is deleted if Corona-Warn-App was the last app using the framework on the device. If you have SwissCovid or Immuni installed in addition to the Corona-Warn-App, the log will only be removed if these additional apps are de-installed or you manually trigger deletion of the exposure log.",
-                            "The way de-installation is handled can change without notice, The same applies to any changes made to the Corona-Warn-App itself. Therefore, only uninstall the app as a last resort to resolve an error.",
+                            "On Android devices, the current implementation <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>deletes the 'database and keys' from the device</a>. Hence, you should expect that the collected random IDs are deleted when the app is uninstalled.",
```
Comment (vaubaehn) [inline src/data/faq.json:410]: To make it more in parallel to the German version, maybe:
On Android devices, the current implementation <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>deletes the 'database and keys' from the device</a> `when uninstalling the app`.

Comment (vaubaehn) [reply src/data/faq.json:410]: or maybe even:
On Android devices, the current implementation `of the Exposure Notification Framework` [deletes the 'database and keys' from the device](https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods) `when uninstalling the app`.

File: src/data/faq_de.json [hunk 2]
```diff
@@ -406,8 +406,8 @@
...
-                            "Auf Android-Geräten werden die Begegnungsaufzeichnungen gelöscht, falls die Corona-Warn-App die letzte App war, die das ENF nutzt. Falls sie zusätzlich z.B. SwissCovid oder Immuni installiert haben, bleiben die Begegnungsaufzeichnungen erhalten, bis auch diese Apps gelöscht werden oder Sie händisch eine Löschung der Begegnungsaufzeichnungen veranlassen.",
-                            "Da sich dieses Verhalten bei der App-Deinstallation auch ohne Änderungen an der Corona-Warn-App jederzeit ändern kann, empfehlen wir, wirklich nur in Ausnahmefällen die App zu löschen.",
+                            "Auf Android-Geräten werden derzeit beim Deinstallieren <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>'Datenbank und Schlüssel' vom Gerät gelöscht</a>. Sie sollten daher davon ausgehen, dass mit Löschung der App auch die Begegnungsaufzeichnungen verloren gehen.",
```
Comment (vaubaehn) [inline src/data/faq_de.json:409]: Accordingly:
Auf Android-Geräten werden derzeit beim Deinstallieren <a href='https://developers.google.com/android/exposure-notifications/exposure-notifications-api#methods' target='_blank' rel='noopener noreferrer'>'Datenbank und Schlüssel' `durch das Exposure Notification Framework` vom Gerät gelöscht</a>.

File: src/data/faq.json (hunk 1 above)
Comment (vaubaehn) [reply src/data/faq.json:410]: Take care: in my review the second line (411) seems to be deleted? That was not my intention, should stay in its place! sorry for any confusion.

File: src/data/faq_de.json (hunk 2 above)
Comment (vaubaehn) [reply src/data/faq_de.json:409]: Same here: in my review the second line (410) seems to be deleted? That was not my intention, should stay in its place! sorry for any confusion.

Comment (tkowark) [reply src/data/faq_de.json:409]: No worries, the diff view in Github is sometimes a bit confusing. I get your point :)

**Pull request merged**